src/carousel/carousel.spec.ts

```diff
@@ -1145,6 +1189,7 @@ class TestComponentOnPush {}
 	template: '',
 })
 class TestComponent {
+	@ViewChild(NgbCarousel, { static: true }) carousel: NgbCarousel;
```

Comment:
best to use the new `viewChild.required` here?
https://angular.dev/guide/components/queries